eventserver/server_billable_handler_test.go

```diff
@@ -96,9 +96,10 @@ var _ = Describe("BillableEventsHandler", func() {
 		Expect(res.Header().Get("Content-Type")).To(Equal("application/json; charset=UTF-8"))
 	})
 
-	It("should require admin scope (for now)", func() {
+	It("should require admin scope or manager role", func() {
```

Comment:
Actually the test check "admin scope or billing access permissions" or  "admin scope or billing access permissions or org manager"

Reply:
Maybe. I though of it like `The user has either (billing or organisation) manager role`.

Reply:
sure, I meant make it more meaningful